### PR TITLE
BDOG-1385: Adds Redirect to Services as default filter

### PR DESCRIPTION
**Pull request by Tyrpix**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -24,7 +24,8 @@ lazy val microservice = Project(appName, file("."))
     RoutesKeys.routesImport ++= Seq(
         "uk.gov.hmrc.cataloguefrontend.connector.model.TeamName",
         "uk.gov.hmrc.cataloguefrontend.model.Environment",
-        "uk.gov.hmrc.cataloguefrontend.shuttering.ShutterType"
+        "uk.gov.hmrc.cataloguefrontend.shuttering.ShutterType",
+        "uk.gov.hmrc.cataloguefrontend.healthindicators.RepoType"
```
Comment (christopherjturner) [inline build.sbt:28]: dont think this is needed now

File: app/uk/gov/hmrc/cataloguefrontend/healthindicators/HealthIndicatorsController.scala [hunk 2]
```diff
@@ -41,20 +41,14 @@ class HealthIndicatorsController @Inject()(
...
-          case None     => repoRatingsWithTeams
-        }
-      } yield
-        filteredIndicators match {
-          case repoRatingsWithTeams => Ok(HealthIndicatorsLeaderBoard(repoRatingsWithTeams, form, repoTypes))
-        }
+      RepoType.parse(repoType).fold(_ => Future.successful(Redirect(routes.HealthIndicatorsController.indicatorsForRepoType(RepoType.Service.asString))),
+        r => for {
+          repoRatingsWithTeams <- healthIndicatorsService.findRepoRatingsWithTeams(Some(r))
+          form = HealthIndicatorsFilter.form.bindFromRequest
```
Comment (cjwebb) [inline app/uk/gov/hmrc/cataloguefrontend/healthindicators/HealthIndicatorsController.scala:49]: Is this form needed now we get repoType as a query param?

Comment (Tyrpix) [reply app/uk/gov/hmrc/cataloguefrontend/healthindicators/HealthIndicatorsController.scala:49]: I think so. When removed the search stopped working correctly

File: app/views/HealthIndicatorsLeaderBoard.scala.html [hunk 3]
```diff
@@ -16,7 +16,8 @@
 
 @import uk.gov.hmrc.cataloguefrontend.healthindicators.{RepoRatingsWithTeams, RepoType}
 @(  repoRatings :   Seq[RepoRatingsWithTeams]
-,   form        :   Form[_]
+,   repoType : RepoType,
+    repositoryName : String
```
Comment (cjwebb) [inline app/views/HealthIndicatorsLeaderBoard.scala.html:20]: Fix the formatting here, please

**Review by cjwebb — APPROVED** [review]

**Review by christopherjturner — APPROVED** [review]

**Pull request merged**